library/src/scala/runtime/LazyVals.scala

```diff
@@ -27,17 +27,18 @@ object LazyVals {
 
   private val base: Int = {
     val processors = java.lang.Runtime.getRuntime.nn.availableProcessors()
-    8 * processors * processors
+    val rawSize = 8 * processors * processors
```

Comment:
not sure why it's `8` here.

Reply:
Some CPU has 384 cores now.

Reply:
maybe 2 * 2 is better than 8

Reply:
I see from blame that it is here for some time. But `availableProcessors` gives the number of logical cores, so no relation to CPU count, just the actual cores. But I don't think this code is actually used in the new lazy vals impl?